### PR TITLE
ADCM-1550 Static config check

### PR DESCRIPTION
**Pull request by grey-kristy**

### CODE REVIEW

File: python/cm/adcm_schema.yaml [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+
+component_item:
+  match: one_of
+  variants:
+    - none
+    - component_dict
+
+component_dict:
+  match: dict
+  items:
```
Comment (skhomuti) [inline python/cm/adcm_schema.yaml:100]: https://github.com/arenadata/docs_adcm/pull/11/files need to add `bound_to` property

File: python/cm/adcm_schema.yaml [hunk 2]
```diff
@@ -0,0 +1,617 @@
...
+    min: version_rule
+    max: version_rule
+    min_strict: version_rule
+    max_strict: version_rule
+
+states_dict:
+  match: dict
+  items:
+    available: available_state
+    on_success: string
```
Comment (skhomuti) [inline python/cm/adcm_schema.yaml:165]: it's right? now there are tests for different types of `on_success` property

File: python/cm/adcm_schema.yaml [hunk 3]
```diff
@@ -0,0 +1,617 @@
...
+  <<: *common_object
+  items:
+    <<: *common_object_items
+    upgrade: upgrade_list
+    import: import_dict
+    export: export
+
+host_object:
+  <<: *common_object
+  items:
```
Comment (skhomuti) [inline python/cm/adcm_schema.yaml:61]: hostcomponent may have `upgrade` property

File: python/cm/adcm_schema.yaml [hunk 4]
```diff
@@ -0,0 +1,617 @@
...
+  <<: *config_dict_sub_common
+  items:
+    <<: *config_dict_sub_items
+    default: map_string_string
+
+config_dict_sub_structure:
+  <<: *config_dict_sub_common
+  items:
+    <<: *config_dict_sub_items
+    default: string
```
Comment (skhomuti) [inline python/cm/adcm_schema.yaml:276]: why default for structure is string?

File: python/cm/adcm_schema.yaml (hunk 1 above)
Comment (skhomuti) [reply python/cm/adcm_schema.yaml:100]: also in doc https://docs.arenadata.io/adcm/sdk/config.html#components property `description` is present
where is the fix required?

Comment (grey-kristy) [reply python/cm/adcm_schema.yaml:100]: fixed

File: python/cm/adcm_schema.yaml (hunk 2 above)
Comment (grey-kristy) [reply python/cm/adcm_schema.yaml:176]: yep, that's right

**Comment by skhomuti** [comment]
Following config is really invalid? `There is no required key \"min\" in map`
```
          versions:
             min_strict: 0.1
             max_strict: 4.0
```

File: python/cm/adcm_schema.yaml (hunk 4 above)
Comment (grey-kristy) [reply python/cm/adcm_schema.yaml]: fixed

**Comment by grey-kristy** [comment]
> max_strict

fixed

File: pytest.sh [hunk 5]
```diff
@@ -18,7 +18,7 @@ pip3 install -r requirements-test.txt
 find . -name "*.pyc" -type f -delete
 find . -name "__pycache__" -type d -delete
 { # try
-    pytest tests/ui_tests tests/functional -s -v -n auto --maxfail 30 \
```
Comment (skhomuti) [inline pytest.sh:21]: don't forget return it back before merge

**Comment by acmnu** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @acmnu! Build has been launched!

**Review by acmnu — APPROVED** [review]

**Pull request merged**